modules/dxp/apps/sharepoint-soap/sharepoint-soap-repository/src/main/java/com/liferay/sharepoint/soap/repository/connector/internal/util/test/SharepointConnectionTestUtil.java

```diff
@@ -80,9 +79,20 @@ private static String _getSharepointVMHostname() throws Exception {
 		try (BufferedReader bufferedReader = new BufferedReader(
 				new InputStreamReader(httpURLConnection.getInputStream()))) {
 
-			Stream<String> payloadStream = bufferedReader.lines();
+			StringBundler sb = new StringBundler();
 
-			String payload = payloadStream.collect(Collectors.joining());
+			String line = null;
+
+			while ((line = bufferedReader.readLine()) != null) {
+				sb.append(line);
+				sb.append(StringPool.BLANK);
```

Comment:
@jeffwu0724 `Stream<String>.collect(Collectors.joining())` just concats all the strings, so here the `StringPool.BLANK` is not necessary.

You can verify it with a simple piece of code:

```
		List<String> strings = Arrays.asList("A", "B", "C");

		System.out.println(strings.stream().collect(Collectors.joining()));
```

outputs `ABC` not `A B C`.